mixins/form-validation.js

```diff
@@ -0,0 +1,109 @@
+import { getAllValues } from '@/utils/object';
+import formRulesGenerator from '@/utils/validators/formRules/index';
+
+export default {
+  data() {
+    return {
+      formRulesets: [
+        /* formRulesets define the validation rules for the entire form. These should almost always be overridden in the form-component using the mixin
+        example:
+        {
+          path:  'spec.template.spec.containers.securityContext.runAsUser', // required: defines the path of the value to be tested against it's rules. Uses this.value unless overridden via optional rootObject
+          rootObject: { container: { image: 'name' } } // optional: redirects the path to the object passed here,
+          rules: ['noSpaces', 'noPeriods'], required: array of functions to run against the value of the field defined by the path (and optionally the rulesets rootObject),
+          translationKey: '', // optional: defines the displaykey, overrides displaykeys that may otherwise be passed into the translation
+        }
+        */
+      ],
+      boundValidationPaths: [
```

Comment:
Or maybe `boundValidationPaths` could be called `validatedFieldPaths`? (Maybe what's throwing me off is that the concept of bind/bound is a can of worms in Kubernetes)